beanprice/price.py

```diff
@@ -252,16 +262,16 @@ def log_currency_list(message, currencies):
         logging.debug("  {:>32}".format('{} /{}'.format(base, quote)))
 
 
-def get_price_jobs_at_date(entries,
-                           date=None,
-                           inactive=False,
-                           undeclared_source=None):
+def get_price_jobs_at_date(entries: List[data.Directive],
+                           date: Optional[datetime.date]=None,
+                           inactive: bool=False,
+                           undeclared_source: Optional[str]=None):
     """Get a list of prices to fetch from a stream of entries.
```

Comment:
The conventional style is to have spaces on both side of the `=` when declaring default values with types.

Reply:
Added space around `=` in those cases as suggested.